components/sidebar-navigation/Components/LinkItem.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { cnb } from "cnbuilder";
+
+/**
+ * Link Item for Sidebar Navigation
+ *
+ * @returns {JSX.Element}
+ * @constructor
+ */
+export default function LinkItem({ level, url, shortName, active }) {
+  return (
+    <a
+      className={cnb(
+        "su-block su-border-l-5 hocus:su-underline hocus:su-text-digital-red hocus:su-border-digital-red dark:hocus:su-text-dark-mode-red dark:hocus:su-border-dark-mode-red",
+        active
```

Comment:
Another good attribute to add for the active link would be `aria-current="page"`
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current

Reply:
Love it, thanks!